recipes/tt-metalium/meta.yaml

```
@@ -0,0 +1,80 @@
+{% set name = "tt-metalium" %}
+{% set version = "0.56.0.rc1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tenstorrent/tt-metal/releases/download/v{{ version.replace(".rc", "-rc") }}/tt-metalium.tar.gz
+  sha256: 319f8c8f5b3ce98ef7db0c0e230cd37aca339b06ece98d923daadf4d86b27465
+
+# Skipping build for other python versions due to timeout in staging CI
+# Will enable more python versions after merge
+build:
+  number: 0
+  skip: true  # [not linux or py!=310]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - cmake
+    - ninja
+    - git
+
+  host:
+    - python
+    - pip
+    - wheel
+    - setuptools
+    - numactl
+    - libhwloc
+    - mpc
+    - numpy
+    - loguru
+    - networkx
+    - pytorch
+    - python-graphviz
+
+  run:
+    - python
+    - numactl
+    - libhwloc
+    - libzlib
+    - mpfr
+    - gmp
+    - mpc
+    - numpy
+    - loguru
+    - networkx
+    - pytorch
+    - python-graphviz
+
+test:
+  imports:
+    - ttnn
+
+about:
+  home: https://github.com/tenstorrent/tt-metal
+  summary: 'Simple, fast, extensible runtime libraries for Tenstorrent Hardware'
+  description: |
+    TT-NN operator library, and TT-Metalium low level kernel programming model.
+  license: Apache-2.0
```

Please update this field with the correct [SPDX](https://spdx.org/licenses/) identifier for the other licenses (deduplicated)
```suggestion
  license: Apache-2.0 AND GPL-2.0-or-later AND ...
```